package Methods;

public class Exception1 {
	public static void main(String[] args)
	{
		try(a/b);{
	         
			int num = 100/0;
			System.out.println("Divided by zero"+num);
		}
		catch(ArithmeticException e)
		{
			e.printStackTrace();
		}
		finally
		{
			System.out.println("Continue Rest code");
		}
		
	}
}

